docs/claim-transparency.md

```diff
@@ -320,3 +317,33 @@ shown in the following example.
   }
 }
 ```
+
+## Related work and standards
+
+The concepts used and introduced in this repository rely heavily on the in-toto standard, and the
+SLSA framework. However, there are other similar efforts in the open-source and security community
+that we try to be consistent with. The most notable of these are
+[SCITT](https://github.com/ietf-scitt) and RATS.
+
+### Comparison to RATS
+
+The Remote ATtestation procedureS (RATS) working group has provided an
+[architecture](https://datatracker.ietf.org/doc/html/draft-ietf-rats-architecture) and glossary of
+concepts related to remote attestation.
+[This cheatsheet](https://github.com/thomas-fossati/rats-cheatsheet) and
+[this slides deck](https://confidentialcomputing.io/wp-content/uploads/sites/85/2021/09/IETF-Remote-Attestation-Architecture-Overview.pdf)
+give an overview of the architecture and the main concepts. RATS has many concepts similar to the
+ones in our design, but seems to be focused on claims and evidence that are generated and consumed
+automatically. Claims and evidence in RATS are designed to be used for remote attestation. The
+claims in our binary transparency ecosystem, however, are not limited to the ones used for remote
+attestation. We target a wider range of use cases.
```

Comment:
Could be nice to give an example of a use case we target which RATS does not.

Reply:
Thanks. Done.